tools/patcher/README.md

```diff
@@ -1,33 +1,47 @@
-# HOW TO PATCH YOUR allui.mrg
+# HOW TO PATCH YOUR allui.mrg OR allpac.mrg
 Well hello there. This tool right here *should* help you in creating your
 own nice allui.mrg + allui.hed to put into your emulator's mod folder.
 
+## 0. Set up a linux environment
+
+If you're on linux, great! Skip this step.
+
+If you're on Windows, the best way is to install Windows subsystem for Linux (WSL)
+https://docs.microsoft.com/en-us/windows/wsl/install
+
 ## 1. Download a few tools.
 
-Install Python (Windows installs here https://www.python.org/downloads/windows/)
+Install python (``sudo apt python`` or your distro equivalent)
 
-Into the **quickbms** folder you need to download quickbms: http://aluigi.altervista.org/quickbms.htm -
-this wonderful swiss army knife tool extracts and rebuilds .mrg files.
+You need Ross's mangetsu toolset: https://github.com/rschlaikjer/mangetsu - get it and build it according
+to the readme there.
 
-Then, for Linux, download compressonatorcli into the **compressonator** folder: https://github.com/GPUOpen-Tools/compressonator/releases/tag/V4.2.5185 -
-this tools builds .dds files from .png files to be inserted. **ALSO**, edit the compressonatorcli script and remove the first empty line before ``#!/bin/bash``.
+Download compressonatorcli: https://github.com/GPUOpen-Tools/compressonator/releases/tag/V4.2.5185 - and rename the ``compressonatorcli`` script to ``compressonator``.
+This tools builds .dds files from .png files to be inserted. **ALSO**, edit the script and remove the first empty line before ``#!/bin/bash``.
 
-For Windows, just install the CompressonatorCLI software, this script uses the default install path.
+The binaries need to be in your PATH system variable for it to work. To do that, edit your ``~/.bashrc``
+file and add them at the end of the file, like so:
```

Comment:
Maybe just add the entire build folder to $PATH instead of having to do it one by one?

Reply:
Hmm you're right, reading a bit more about all that PATH adding now and that should work better.

Reply:
@rschlaikjer should be better now.